### PR TITLE
doc: add minutes for 3 Sep 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/tsc need at least one reviewer.

File: meetings/2020-09-03.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ 
+* Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+
+### nodejs/node
+
+
+* \[v14.x backport\] stream: simpler and faster Readable async iterator  [#34887](https://github.com/nodejs/node/pull/34887)
+  * Please review/comment on the issue
+  * Myles, see Matteo chiming that it is a good thing so comfortable that it will land, personally
```
Comment (MylesBorins) [inline meetings/2020-09-03.md:41]: oops 😂

**Review by MylesBorins — APPROVED** [review]
LGTM

File: meetings/2020-09-03.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  * no progress
+
+## Strategic Initiatives
+
+* build - no update, 
+* Modules - open discussion on Node.js/node on module export patterns, don’t have consensus
+  if it should land - https://github.com/nodejs/node/pull/34718, would like to get feedback from
+  TSC members.
+* Future of build chain, repository created, issues starting to be populated.
+* next tens years of Node.js - please take a look if you have a chance https://github.com/nodejs/next-10/pull/11/files
```
Comment (nschonni) [inline meetings/2020-09-03.md:86]:
```suggestion
* next ten years of Node.js - please take a look if you have a chance https://github.com/nodejs/next-10/pull/11/files

```

**Pull request merged**